tools/depends/configure.ac

```diff
@@ -296,6 +312,11 @@ case $host in
     platform_cxxflags="$platform_cflags"
     platform_os="linux"
     ffmpeg_options_default="--enable-vaapi --enable-vdpau --cpu=$use_cpu"
+    if [[ $host =~ "i686"*"-linux-gnu"* ]] || [[ $host =~ "i"*"86"*"-"*"-linux-uclibc"* ]]; then
+      use_cmake_cpu=i686
+    else
+      use_cmake_cpu=x86_64
```

Comment:
There is a variable which already contains exactly one of the two values, depending on the host used to configure. So why not use that one?

Reply:
I look about a regex (sed) to extract it.

Reply:
Thought you mean the `$host` and not come direct to them that `$host_cpu` is created from them.

But why we have so crazy switch compares about `$host` :smile:? 
A `*i686*-linux-gnu*` or `x86_64*-linux-gnu` say me that something can before or behind host cpu and not only a i686 possible :thinking: